### PR TITLE
User/getrou/release pipeline fix

### PR DESCRIPTION
**Pull request by getrou**
Update the release pipeline to not fail now that MRTCore is in the ProjectReunion nuget package.

### CODE REVIEW

File: build/ProjectReunion-release.yml [hunk 1]
```diff
@@ -72,9 +72,9 @@ jobs:
     doCodeSign: 'true'
 
 - job: SignBinariesAndPublishSymbols
-  dependsOn: Build
+  dependsOn: PublishMRT
```
Comment (huichen123) [inline build/ProjectReunion-release.yml:75]: >PublishMRT [](start = 13, length = 10)

should also depend on Build

File: build/SignConfig.xml [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
-    <file src="__INPATHROOT__\Release\arm\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\arm\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
-    <file src="__INPATHROOT__\Release\arm\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" signType="WindowsRT" dest="__INPATHROOT__\Release\arm\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" />
-    <file src="__INPATHROOT__\Release\arm64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\arm64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
-    <file src="__INPATHROOT__\Release\arm64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" signType="WindowsRT" dest="__INPATHROOT__\Release\arm64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" />
+    <file src="__INPATHROOT__\runtimes\win10-x86\native\Microsoft.ApplicationModel.Resources.dll" signType="WindowsRT" dest="__INPATHROOT__\runtimes\win10-x86\native\Microsoft.ApplicationModel.Resources.dll" />
```
Comment (huichen123) [inline build/SignConfig.xml:5]: ><file [](start = 4, length = 5)

compare with Singconfig.xml for MRT, it misses a few files.

Comment (EHO-makai) [reply build/SignConfig.xml:5]: right, looks like this needs mrm.dll at minimum.  Looks good overall, though, thanks.

Comment (getrou) [reply build/SignConfig.xml:5]: We don't need to sign the lib or pdb files correct?

Comment (EHO-makai) [reply build/SignConfig.xml:8]: yes, that is correct, just dll and winmd.

Comment (EHO-makai) [reply build/SignConfig.xml:8]: @huichen123, is mrm.dll supposed to be in this list?

File: build/ProjectReunion-release.yml (hunk 1 above)
Comment (getrou) [reply build/ProjectReunion-release.yml:75]: I believe Build just builds the Project Reunion sample code project that we aren't using anymore.  We aren't signing or publishing any of the output of Build in signconfig.xml I think, so we should be good.

Comment (huichen123) [reply build/ProjectReunion-release.yml:75]: yes, it's ok for now, but it won't be always empty. :)

File: build/SignConfig.xml (hunk 2 above)
Comment (huichen123) [reply build/SignConfig.xml:8]: yes, all dlls need signed. still missing winmd and Microsoft.ApplicationModel.Resources.Projection.dll.

File: build/ProjectReunion-release.yml (hunk 1 above)
Comment (getrou) [reply build/ProjectReunion-release.yml]: Okay, that makes sense.  I'll have it depend on both.

File: build/SignConfig.xml (hunk 2 above)
Comment (getrou) [reply build/SignConfig.xml:8]: It should have all the files now, the winmd and both copies of the projection dlls.

**Review by huichen123 — APPROVED** [review]

**Review by EHO-makai — APPROVED** [review]
Make sure mrm.dll is in the signconfig.xml before completing the PR, thanks. :-)

**Pull request merged**